frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { riskIcons, colorScale, legendColorScale, riskSorting, mapMetrics } from './mappers';
+import { PrometheusHealthPopupProps } from '@console/plugin-sdk';
+import './style.scss';
+
+const DataComponent: React.FC<DataComponentProps> = ({ x, y, datum }) => {
+  const Icon = riskIcons[datum.id];
+  return <Icon x={x} y={y - 5} fill={legendColorScale[datum.id]} />;
+};
+
+export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses, k8sResult }) => {
+  const resource = mapMetrics(responses[0].response);
+  const clusterId = _.get(k8sResult, 'data.spec.clusterID', '');
+  const riskEntries = Object.entries(resource).sort(
+    ([k1], [k2]) => riskSorting[k1] - riskSorting[k2],
+  );
+  const numberOfIssues = Object.values(resource).reduce((acc, cur) => acc + cur, 0);
+  const isThereIssues = riskEntries.length > 0 && numberOfIssues > 0;
+
+  return (
+    <div className="co-insights__box">
+      <div className="co-status-popup__section">
+        Insights identifies and prioritizes risks to security, performance, availability, and
+        stability of your clusters.
+      </div>
+      <div className="co-status-popup__section">
+        {isThereIssues && (
+          <div>
+            <ChartDonut
+              data={riskEntries.map(([k, v]) => ({
+                label: `${v} ${k}`,
+                x: k,
+                y: v,
+              }))}
+              title={`${numberOfIssues}`}
+              subTitle="Total issues"
+              legendData={Object.entries(resource).map(([k, v]) => ({ name: `${k}: ${v}` }))}
+              legendOrientation="vertical"
+              width={300}
+              height={150}
+              colorScale={colorScale}
+              constrainToVisibleArea
+              legendPosition="left"
+              legendComponent={
+                <ChartLegend
+                  title="Total Risk"
+                  titleComponent={
+                    <ChartLabel
+                      style={{ fontWeight: 'bold', fontSize: 'var(--pf-c-title--m-md--FontSize)' }}
+                    />
+                  }
+                  data={riskEntries.map(([k, v]) => ({
+                    name: `${v} ${k}`,
+                    id: k,
+                  }))}
+                  dataComponent={<DataComponent />}
+                />
+              }
+              padding={{
+                bottom: 20,
+                left: 145,
+                right: 20, // Adjusted to accommodate legend
+                top: 0,
+              }}
+            />
+          </div>
+        )}
+        {!isThereIssues && (
+          <div className="co-insights__no-rules">No Insights data to display.</div>
+        )}
+      </div>
+      <div className="co-status-popup__section">
+        {isThereIssues && (
+          <>
+            <div style={{ fontWeight: 'bold' }}>Fixable issues</div>
+            <div>
+              <a href={`https://cloud.redhat.com/openshift/details/${clusterId}`}>
```

Comment:
Use the `ExternalLink` component. Here's an example:

https://github.com/openshift/console/blob/24912acc536dd84b8e9e3317fc3a324a2cbf8bfd/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-community-provider-modal.tsx#L52-L55